### PR TITLE
feat(elb): add new check `elb_is_in_multiple_az`

### PR DESCRIPTION
**Pull request by puchy22**
### Context

Checks that Classic Load Balancers are configured with registered instances from at least the specified number of Availability Zones (AZs). The control fails if the load balancer does not span the minimum number of AZs specified.

### Description

- [x]  Modify service to add AZs to ELB
- [x]  Test service changes
- [x]  Add configurable parameter for the check
- [x]  Add check logic
- [x]  Add unit testing for the check

### Checklist

- Are there new checks included in this PR? Yes
    - If so, do we need to update permissions for the provider? No.
- [x] Review if the code is being covered by tests.
- [x] Review if code is being documented following this specification https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: prowler/providers/aws/services/elb/elb_is_in_multiple_az/elb_is_in_multiple_az.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+from typing import List
+
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.elb.elb_client import elb_client
+
+# https://www.trendmicro.com/cloudoneconformity/knowledge-base/aws/ELB/ec2-instances-distribution-across-availability-zones.html
+# https://docs.aws.amazon.com/securityhub/latest/userguide/elb-controls.html#elb-10
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/elb/elb_is_in_multiple_az/elb_is_in_multiple_az.py]: Please, remove this.

File: prowler/providers/aws/services/elb/elb_is_in_multiple_az/elb_is_in_multiple_az.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        findings = []
+        ELB_MIN_AZS = elb_client.audit_config.get("elb_min_azs", 2)
+        for loadbalancer_arn, load_balancer in elb_client.loadbalancers.items():
+            report = Check_Report_AWS(self.metadata())
+            report.region = load_balancer.region
+            report.resource_id = load_balancer.name
+            report.resource_arn = loadbalancer_arn
+            report.resource_tags = load_balancer.tags
+            report.status = "FAIL"
+            report.status_extended = f"Classic Load Balancer {load_balancer.name} is not in at least {ELB_MIN_AZS} availability zones. Is only in {', '.join(load_balancer.availability_zones)}."
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/elb/elb_is_in_multiple_az/elb_is_in_multiple_az.py]:
```suggestion
            report.status_extended = f"Classic Load Balancer {load_balancer.name} is not in at least {ELB_MIN_AZS} availability zones, it is only in {', '.join(load_balancer.availability_zones)}."
```

File: prowler/providers/aws/services/elb/elb_is_in_multiple_az/elb_is_in_multiple_az.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+            if len(load_balancer.availability_zones) >= ELB_MIN_AZS:
+                report.status = "PASS"
+                report.status_extended = f"Classic Load Balancer {load_balancer.name} is in {len(load_balancer.availability_zones)} availability zones. Currently in {', '.join(load_balancer.availability_zones)}."
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/elb/elb_is_in_multiple_az/elb_is_in_multiple_az.py]:
```suggestion
                report.status_extended = f"Classic Load Balancer {load_balancer.name} is in {len(load_balancer.availability_zones)} availability zones: {', '.join(load_balancer.availability_zones)}."
```

File: prowler/providers/aws/services/elb/elb_service.py [hunk 4]
```diff
@@ -99,4 +100,5 @@ class LoadBalancer(BaseModel):
     access_logs: Optional[bool]
     listeners: list[Listener]
     cross_zone_load_balancing: Optional[bool]
+    availability_zones: Set[str]
```
Comment (jfagoagas) [inline prowler/providers/aws/services/elb/elb_service.py]:
```suggestion
    availability_zones: set[str]
```

File: prowler/providers/aws/services/elb/elb_service.py [hunk 5]
```diff
@@ -1,4 +1,4 @@
-from typing import Optional
+from typing import Optional, Set
```
Comment (jfagoagas) [inline prowler/providers/aws/services/elb/elb_service.py]:
```suggestion
from typing import Optional
```

**Review by jfagoagas — CHANGES_REQUESTED** [review]

**Review by jfagoagas — APPROVED** [review]

**Pull request merged**